src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -18,6 +18,12 @@ export default function CSVFileImport({url, title}: CSVFileImportProps) {
   const classes = useStyles();
   const [file, setFile] = useState<any>();
 
+  useEffect(() => {
+    console.log(process.env.REACT_APP_USERNAME)
+    console.log(process.env.REACT_APP_PASSWORD)
+    localStorage.setItem('authorization_token', Buffer.from(`${process.env.REACT_APP_USERNAME}:${process.env.REACT_APP_PASSWORD}`).toString('base64'));
```

Comment:
All requests are authorized, so it is impossible to test unauthorized access. 
`authorization_token` should be set by a user manually.